### DISCUSSION TITLE
2022-09-16 OMI Champions Meeting

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#292] [General]
draft agenda for the proposed September 16th 19:00 UTC (noon Pacific) OMI champions follow-up meeting (re: https://github.com/omigroup/omigroup/discussions/288)

**Comment by humbletim** [comment]
### draft definitions 
| distinction                           | description |
|----------------------------|---------------------------------------------------------------------------------------------------------------------------------|
| **Champion**               | _Individuals in the OMIgroup who have been elected to take on leadership roles._                                                |
| **Allies/Fellows/Friends** | _Groups who align with OMIgroup principles._                                                                                    |
| **Sponsors**               | _Contributors to OMIgroup who expect something in return (such as recognition). Event Sponsors only relate to specific events._ |
| **Donors**                 | _Financial supporters of OMIgroup who do not expect anything in return._

**Comment by humbletim** [comment]
proposed agenda points:

- [ ] **Review draft OMI role definitions / last meeting's notes**.
- <details>

  - Formatted markdown table above..
  - Previous meeting agenda/notes: https://github.com/omigroup/omigroup/discussions/288.

</details>

- [ ] **Gather updated champion availabilities, preferred areas of focus, and compare with needs/milestones on the OMI radar.**
- <details>

   - Should we rough-in a Team Working Agreement template? (ie: for the still somewhat abstract "OMI Champion meta-Subgroup").
   - What's the recommended way to address sensitive feedback (ie: CoPC concerns, diplomatic escalations, conflict resolution)?

   - Potentially several different but related meta-governance workflows seem to have emerged:
     - OMI **Org-level** stuff (OMI Group entity status, strategic alliances, legal agreements, etc.).
     - OMI **Back Office** (tools/service integration, tech support, misc. administrative).
     - OMI **Local Community** engagement.
     - OMI **Subgroup Facilitation** (ie: supporting the set of all OMI subgroups, their leads, upstream touchpoints).
     - OMI **Peer Community** engagement (social media? outside community collaborations?).
     - ... ?

</details>

- [ ] **Review OMI Treasury / Budgeting**:
- <details>

  - What's the current process for requesting budget, receiving approval, charging the credit card, etc.?
  - What are our policies related to accepting donations?
    - Are potential donors "required" to align with OMI core principles?
    - How could we clarify donor relationships and expectations?
  - Is there an existing overview report (or could we rough one in) listing all known/recurring OMI expenses?
  - Does anyone know of other (relevant) out-of-pocket costs being paid by individuals right now?
  - ... ?

</details>

- [ ] OMI fiscal development discussions (next steps? subgroup <=> co-champions sync?)

- [ ] https://github.com/omigroup/omigroup/issues/289

**Comment by humbletim** [comment]
- [ ] **Knowledge Sharing: OpenCollective Receipt Management..,**.
- <details>

    - ![image](https://user-images.githubusercontent.com/94753/190692315-9c887a11-c7aa-4e04-8371-9b5e5595a298.png)

</details>

**Comment by humbletim** [comment]
- [ ] Follow-up re inactive champions?

**Reply by humbletim** [reply]
... added to OMI Admin Spreadsheet; TBD pending review

**Comment by humbletim** [comment]
re: non-profit resource requirements:
 - estimated $10000 initial funding to cover legal filing and bookkeeping:
   - $200/month bookkeeper (ideally curate internally but potentially outsourced)
   - charter
   - filing and legal costs
   - treasurer on the board and "knowable to the IRS"
   - meeting minutes
   - long term strategy (may take several months to work across gov't filing steps/approval)
   - ACTION: schedule a workshop (probably with whole community)

**Comment by mrmetaverse** [comment]
I am working through the receipts bug

**Reply by mrmetaverse** [reply]
[Open Metaverse Interoperability Group · Expenses - Open Collective.pdf](https://github.com/omigroup/omigroup/files/9586777/Open.Metaverse.Interoperability.Group.Expenses.-.Open.Collective.pdf) - cc @humbletim for records

**Reply by mrmetaverse** [reply]
[Open Metaverse Interoperability Group Mail - Your 1Password invoice (Open Metaverse Interoperability Group)_.pdf](https://github.com/omigroup/omigroup/files/9586802/Open.Metaverse.Interoperability.Group.Mail.-.Your.1Password.invoice.Open.Metaverse.Interoperability.Group._.pdf)

**Reply by mrmetaverse** [reply]
[Open Metaverse Interoperability Group Mail - Google Workspace_ Your invoice is available for omigroup.org.pdf](https://github.com/omigroup/omigroup/files/9586816/Open.Metaverse.Interoperability.Group.Mail.-.Google.Workspace_.Your.invoice.is.available.for.omigroup.org.pdf)

**Reply by mrmetaverse** [reply]
[Open Metaverse Interoperability Group Mail - Your receipt from RealtimeBoard Inc. dba Miro #2384-5920.pdf](https://github.com/omigroup/omigroup/files/9586821/Open.Metaverse.Interoperability.Group.Mail.-.Your.receipt.from.RealtimeBoard.Inc.dba.Miro.2384-5920.pdf)

**Comment by humbletim** [comment]
re: potential community-community collaborations with others like xrguild
- co-hosting and sponsoring local in-person meetups / events

**Comment by humbletim** [comment]
OMI Admin Responsibilities starting point:
  - https://docs.google.com/spreadsheets/d/1fcjNFsb6Ta6nFKNla9HhaWqOhLi4BA5PKcSYFdCejko/edit#gid=0
  - 
reference points:
  - #211
  - #165 
  - #32

**Comment by humbletim** [comment]
re: dework
- rough notes overhead @madjin
  - option to use discord/token/role to manage and administrate dwork tasks
  - potential $500-$1000 budget allocation could make sense to run experiment with one workgroup group as a trial?
  - low hanging fruit might be lightweight tasks:
    - notetaking during meetings
    - graphics/badges 
    - changelogs/summaries

- humbletim questions:
  - https://gnosis-safe.io as a way to achieve routine consensus across (elected? champions? budget managers?)
  - how would literal budget allocations flow start-to-finish through proposed workflow?
  - what steps are involved (and which stakeholders are necessary/optional) in order to route through the system?
  - https://coordinape.com/ -- "solution to contributor rewards, feedback, and all things people"
  - would it be possible to purely practice/prototype as a first step? (ie: intentionally avoid high risk tasks or creating tangential burdens on ongoing existing efforts (until proven out))
